app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,97 +1,127 @@
+        <MenuBar.Block label="Asia" count={60} />
 
-  if (loading) {
-    return <PageLoader />;
-  }
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: () => <Search size={20} />
+            },
 
-  return (
-    <>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            onClick={() => setShowNewNotePane(true)}
-            label="Add New Note"
-            icon="ri-add-line"
-          />
-        }
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            searchProps={{
-              value: searchTerm,
-              onChange: e => setSearchTerm(e.target.value),
-              clear: () => setSearchTerm("")
-            }}
-            deleteButtonProps={{
-              onClick: () => setShowDeleteAlert(true),
-              disabled: !selectedNoteIds.length
-            }}
-          />
-          <NoteTable
-            selectedNoteIds={selectedNoteIds}
-            setSelectedNoteIds={setSelectedNoteIds}
-            notes={notes}
-          />
-        </>
-      ) : (
-        <EmptyState
-          image={EmptyNotesListImage}
-          title="Looks like you don't have any notes!"
-          subtitle="Add your notes to send customized emails to them."
-          primaryAction={() => setShowNewNotePane(true)}
-          primaryActionLabel="Add New Note"
+            {
+              icon: () => <Plus size={20} />
+            },
+            {
+              icon: () => <Settings size={20} />
+            }
+          ]}
+        >
+          <Typography
+            component="h4"
+            style="h5"
+            textTransform="uppercase"
+            weight="bold"
+          >
+            Tags
+          </Typography>
+        </MenuBar.SubTitle>
+
+        <MenuBar.Search
+          collapse={isSearchCollapsed}
+          onCollapse={() => setIsSearchCollapsed(true)}
         />
-      )}
-      <NewNotePane
-        showPane={showNewNotePane}
-        setShowPane={setShowNewNotePane}
-        fetchNotes={fetchNotes}
-      />
-      {showDeleteAlert && (
-        <DeleteAlert
-          selectedNoteIds={selectedNoteIds}
-          onClose={() => setShowDeleteAlert(false)}
-          refetch={fetchNotes}
+        <MenuBar.Block label="Sales" count={80} />
+        <MenuBar.Block label="Finance" count={60} />
+        <MenuBar.Block label="Asia" count={60} />
+      </MenuBar>
+
+      <div className="flex flex-col content-start justify-items-start w-full p-5">
+        <Header
+          actionBlock={[
+            <Input
+              key="header_search"
+              placeholder="Search Name, Email, Phone Number, Etc."
+              prefix={<Search size={16} />}
+            />,
+            <Button
+              key="header_button"
+              label="Add Note"
+              className="px-2.5 py-2 ml-3"
+              icon={() => <Plus size={18} className="ml-3" />}
+            />
+          ]}
```

Comment:
Please refer [space between](https://tailwindcss.com/docs/space#class-reference) utils. This is a better approach when you want to evenly space blocks.